src/cards/fancycard/DraftIllustration.tsx

```diff
@@ -41,26 +52,32 @@ export const DraftIllustrationUI = observer(function DraftIllustrationUI_(p: {
         )
     }
 
+    const illustration = fileURLToPath(draft.data.illustration)
+
     // 3. show illustration on top
     return (
         <div style={dropStyle} ref={dropRef} className='relative DROP_IMAGE_HANDLER'>
             {p.revealAppIllustrationOnHover ? ( //
-                <div tw='absolute opacity-0 hover:opacity-100' style={{ transition: 'opacity 0.2s' }}>
+                <div tw='absolute opacity-0 hover:opacity-100 z-50 bg-neutral' style={{ transition: 'opacity 0.2s' }}>
                     <AppIllustrationUI app={draft.app} size={p.size} />
                 </div>
             ) : null}
-            <img
-                // onError={(ev) => {
-                // TODO 2024-01-25 rvion: make it so wiping images doesn't break drafts too much
-                // }}
-                className={p.className}
-                loading='lazy'
-                tw={['bg-base-300', 'rounded', p.onClick ? 'cursor-pointer' : null]}
-                style={{ width: p.size, height: p.size, objectFit: 'contain', imageRendering: 'pixelated' }}
-                src={draft.data.illustration}
-                alt='draft illustration'
-                onClick={p.onClick}
-            />
+            {existsSync(illustration) ? (
```

Comment:
do we keep that ? need proper file middleware at some point; we already have the watcher